openff/evaluator/datasets/taproom/taproom.py

```diff
@@ -398,6 +398,8 @@ def _build_metadata(
                 "n_pull_windows": n_pull_windows,
                 "release_windows_indices": [*range(len(attach_lambdas))],
                 "release_lambdas": release_lambdas,
+                "bound_window_index": [[*range(n_pull_windows)][0]],
```

Comment:
Should `bound_window_index` and `unbound_window_index` be lists, or can they just be single values? I think if they can be single values this will simplify `_paprika_build_end_states_protocol` by a fair bit!

Reply:
I set `bound_window_index` and `unbound_window_index` as lists because `template_values` from `ProtocolReplicator` requires a list as input. Are you referring to setting `template_value` and `template_index` instead? https://github.com/openforcefield/openff-evaluator/blob/e55c86b634af32c431ab8cc301ff937b8201fac4/openff/evaluator/workflow/schemas.py#L136-L138